### DISCUSSION TITLE
automation.xml Documentation

### DISCUSSION THREAD

**Discussion by sbrunner** [aeon-php/automation#310] [General]
I didn't find any documentation of the automation.xml, where can I found it?

**Comment by norberttech** [comment]
hey @sbrunner!
so automation.xml was supposed to be a configuration file for the app however it turned out to be more convenient to use arguments/options.
So far I had not removed it as it might come handy when I start working on monorepo support to for example define sub-projects paths.
Are you looking for any particular feature?

**Comment by sbrunner** [comment]
Ok, thanks, I will found if there are some ways to change the types of changes categorization, usually I also work with the categories chore and dependencies...
And I don't know if there is a way to change the type of change with the labels or with the pull request title (e.g. by using [conventional commit](https://www.conventionalcommits.org/) convention in the pull request title)?

**Comment by norberttech** [comment]
Oh I see, yeah so unfortunately at this point it's impossible to change the change categories but I can see how automation.xml could be a good fit for that feature.
If you would be interested in contributing to this project by creating that feature I would be more than happy to provide you some guidance. 
I can also take care of this one but due to my limited availability, end of year, and other projects I'm currently focused on I can't provide any reliable timeline :( 

> And I don't know if there is a way to change the type of change with the labels or with the pull request title (e.g. by using [conventional commit](https://www.conventionalcommits.org/) convention in the pull request title)?

PR Title is not supported, label is not supported, but conventional commits [are supported.](https://github.com/aeon-php/automation#3-analyze-project-history)
